docs/components/material.md

```diff
@@ -498,44 +504,231 @@ void main() {
 
 varying vec2 vUv;
 uniform vec3 color;
-uniform float time;
```

Comment:
`time1000` is a weird name. Let’s keep time. It is already consistent with other time related variables that use miliseconds in the code base

Reply:
So the reason I didn't use `time` is because that is commonly used to represent time, but in seconds not milliseconds.  Unfortunately the time type is processed as milliseconds, so I wanted to pick a different name and convert to the time in seconds that typical sharers expect.  Maybe `timeMsec` or something more descriptive?

Reply:
To clarify, there is inherent support for type `time`, without using a `tick()` handler to update anything, but it is in milliseconds rather than seconds (which is what you generally see in common sharers from Shadertoy, Shaderfrog, etc.)

Reply:
In the DOM and A-Frame we use miliseconds for time. I would keep the name `time`

Reply:
I may not be doing a good job of explaining.

The shader examples that are time-varying already use a uniform named `time` that is in seconds.  Rather than trying to force already working shaders to rename, what I am illustrating is how to do a simple two line addition that allows the shader code to be reused without modification.

Reply:
Changed to `timeMsec`, and added comments in the shader to explain the conversion to seconds.